plume-models/src/lib.rs

```diff
@@ -306,7 +334,9 @@ pub fn ap_url(url: &str) -> String {
 #[cfg(test)]
 #[macro_use]
 mod tests {
-    use diesel::{dsl::sql_query, Connection, RunQueryDsl};
+    #[cfg(feature = "sqlite")]
```

Comment:
does this mean no more warnings when building with pg backend?

Reply:
Normally, yes. :slightly_smiling_face: